src/main/java/org/jabref/gui/ClipBoardManager.java

```diff
@@ -47,15 +48,18 @@
     private static Clipboard clipboard;
     private static java.awt.datatransfer.Clipboard primary;
     private static ImportFormatReader importFormatReader;
+    private final PreferencesService preferencesService;
 
-    public ClipBoardManager() {
-        this(Clipboard.getSystemClipboard(), Toolkit.getDefaultToolkit().getSystemSelection(), Globals.IMPORT_FORMAT_READER);
+    public ClipBoardManager(PreferencesService preferencesService) {
+        this(Clipboard.getSystemClipboard(), Toolkit.getDefaultToolkit().getSystemSelection(), Globals.IMPORT_FORMAT_READER, preferencesService);
     }
 
-    public ClipBoardManager(Clipboard clipboard, java.awt.datatransfer.Clipboard primary, ImportFormatReader importFormatReader) {
+    public ClipBoardManager(Clipboard clipboard, java.awt.datatransfer.Clipboard primary, ImportFormatReader importFormatReader, PreferencesService preferencesService) {
         ClipBoardManager.clipboard = clipboard;
         ClipBoardManager.primary = primary;
         ClipBoardManager.importFormatReader = importFormatReader;
+
```

Comment:
I guees Tobi or Olli will complain about the empty space ;)
```suggestion
```

Reply:
No here it's fine...just don't add empty lines at the beginning or end of a method